### ISSUE TITLE
Deploying with CSAR.

### ISSUE THREAD

**Issue by chinmaya-dehury** [radon-h2020/xopera-opera#10]
Hi, 
I am working with TOSCA modelling of data pipelines. I have prepared a service template using Winery. The service template consists of creating an instance in private cloud (OpenStack Environment) and installing Apache Nifi atop that instance. I exported the service template in CSAR format.  

Now trying to test my model with xopera using following command  
`opera deploy NifiEnv_deployTest.csar` 

This one is throwing  an error with the following ending line
`UnicodeDecodeError: 'utf-8' codec can't decode byte 0xbb in position 12: invalid start byte` 

Where could be the error? 
Please let me know if you need any further detailed information. 

-Thanks

**Comment by anzoman** [comment]
Hi,

right now xOpera doesn't support compressed/zipped CSAR files but I think that this will be available very soon (you can see the progress on that in this issue: https://github.com/xlab-si/xopera-opera/issues/27) and when this will be fully developed I will let you know here. So in the meantime instead of that you can try to deploy your solution by providing path to the TOSCA service template file (for instance with command `opera deploy service-template.yml`).

I think that it would also be helpful if you take a look at the very first opera documentation here: https://xlab-si.github.io/xopera-docs/

**Comment by chinmaya-dehury** [comment]
Thanks @anzoman for the info. Currently, I am providing YAML file as the argument to opera ([v0.5.2](https://github.com/radon-h2020/xopera-opera/releases/tag/0.5.2)) deploy command.

**Comment by anzoman** [comment]
This issue was addressed in #55. Now opera supports deploying/undeploying service templates and also the compressed CSAR files. This was achieved by implementing `opera init` command and the deployment can be done as follows:

```console
# initialize the CSAR (from the prepared compressed CSAR file) 
# also provide inputs
opera init --inputs inputs.yaml test.csar

# deploy the initialized CSAR
opera deploy

# uddeploy the deployed CSAR
opera deploy
```

So I guess that this is it - we can close the issue!
